app.js

```diff
@@ -0,0 +1,73 @@
+
+'use strict';
+
+var port = process.env.PORT || 3000;
+var express = require('express');
+var Home = require('./models/home');
+var Venue = require('./models/venue');
+
+var app = express();
+
+/* 
+ * MIDDLEWARE
+ * serve images from the public directory
+ */
+app.use(express.static('./public/img/', {
+	setHeaders: function(res, path, stat) {
```

Comment:
I don't think you'll need the `setHeaders` config – as far as I know `express.static` detects it based on the file extension :)